concent_api/concent_api/system_check.py

```diff
@@ -252,7 +252,10 @@ def check_atomic_requests(app_configs = None, **kwargs):  # pylint: disable=unus
     errors = []
     if hasattr(settings, 'DATABASES') and isinstance(settings.DATABASES, dict):
         for database_name, database_config in settings.DATABASES.items():
-            if 'ATOMIC_REQUESTS' not in database_config or database_config['ATOMIC_REQUESTS'] is not True:
+            if (
+                database_config.get('ENGINE') != 'django.db.backends.dummy' and
+                ('ATOMIC_REQUESTS' not in database_config or database_config['ATOMIC_REQUESTS'] is not True)
```

Comment:
You could simplify this line to: `database_config.get('ATOMIC_REQUESTS', False)`

Reply:
Actually your way is not working exactly the same, because it returns True for `database_config['ATOMIC_REQUESTS']` values like `'a'`, `1` etc. I think we should check explicitly for `True`.

Reply:
Yeah, but that's exactly why I personally like to treat booleans differently in `if`s.

In `if variable is None:` vs `if variable:` I prefer the former because the latter is more prone to errors. It's likely that you don't want to treat zero as `None` and if it's not stated explicitly you have to always stop and think about it when reading the code.

In `if variable is True:` vs `if variable:` I prefer the latter because there's no such problem here. If I know the value is either `False` or `True` then it's fine. If I'm worried that it's something other than that this means that there's an `assert` or a validation missing somewhere. And even if for whatever reason it turns out that the value is not boolean, the rules are such that the result will still most likely be sensible. Treating `'a'`, `1` as `True` is more likely to be correct than treating them as `False`.

So the fact that in my version these values are treated differently is a feature, not a bug :)

Reply:
Simply, if you're worried about that I think you should just add an explicit check for `isinstance(database_config['ATOMIC_REQUESTS'], bool)`.

Reply:
But in your version the condition will be met when `ATOMIC_REQUESTS` will be set in `database_config`, which will result in raising error, and we want the opposite.

Reply:
Yeah, sorry, I missed that it's `is not True`, rather than `is True`. The point still stands.

I'd write it like this:
``` python
if database_config.get('ENGINE') != 'django.db.backends.dummy' and not database_config.get('ATOMIC_REQUESTS', False):
     errors.append(create_error_19_atomic_requests_not_set_for_database(database_name))
```

Or, if I wanted to be sure that the value is definitely a boolean:
``` python
if database_config.get('ENGINE') != 'django.db.backends.dummy':
    atomic_requests = database_config.get('ATOMIC_REQUESTS', False)

    if not isinstance(atomic_requests, bool):
         errors.append(create_error_18_invalid_setting_type(f"DATABASES[{database_name}]['ATOMIC_REQUESTS']", atomic_requests))

    if not atomic_requests:
         errors.append(create_error_19_atomic_requests_not_set_for_database(database_name))
```

Reply:
:+1: